app/controllers/messages_cotroller.rb

```diff
@@ -0,0 +1,11 @@
+class MessagesController < ApplicationController
+    def create
+      @message = current_user.messages.create(msg_params)
+    end
+  
+    private
+  
+    def msg_params
+      params.require(:message).permit(:content, :room_id)
+    end
+end
```

Comment:
Add a default new line at the end of the file to your Visual studio ;)
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

Reply:
Also set the indent to the 2 spaces, instead of 2 tabs